sufia-models/app/models/concerns/sufia/generic_file/csv.rb

```diff
@@ -0,0 +1,23 @@
+module Sufia
+  module GenericFile
+    module CSV
+      def to_csv
+        ::CSV.generate do |csv|
+          csv << csv_terms.map do |term|
+            values = send(term)
+            values = [values].flatten # make sure we have an array
+            values.join(multi_value_separator)
```

Comment:
@mjgiarlo I would think it would be hard to make a consistent table with multiple files if one row has 3 values and the next only has two.  @mjgiarlo How is @jcoyne's importer handling that?  Is is he determining the maximum and having all rows the same?  for example I have one file with 2 contributors and one file with 1.  How does that show up in the file?